scss/tech.scss

```diff
@@ -0,0 +1,175 @@
+.parent-ticket {
+    display: flex;
+    flex-flow: row wrap;
+    margin: 5%;
+  
+    .ticket {
+      .outer-tech {
+        border: 1px solid $primary-green;
+        border-radius: 15px;
+        padding: 10px;
+        margin: 30px;
+        width: 250px;
+        height: 380px;
+        transition: all .5s ease;
+  
+          &:hover {
+            transform: translate(30px,30px);
+          }
+  
+          .tech {
+            position: absolute;
+            width: 270px;
+            height: 400px;
+            background: $primary-green;
+            display: flex;
+            flex-direction: column;
+            justify-content: space-evenly;
+            align-items: center;
+            border-radius: 15px;
+            transition: all .5s ease;
```

Comment:
i think you can be more specific with the kind of transition. 'all' is so general